chat-commands.js

```diff
@@ -2858,7 +2925,7 @@ exports.commands = {
 
 		Ladders.disabled = true;
 
-		this.logModCommand(`The ladder was disabled by ${user.name}.`);
+		this.modlog(`DISABLELADDER`);
```

Comment:
I think this should still modlog who disabled it

Reply:
it does
``this.modlog('ID')``; shows as ``ID: by this.user.userid``